i18n/transifex.py

```diff
@@ -46,8 +46,7 @@ def push_all():
 
 def pull(configuration, *resources):
     """
-    Pull translations from all languages listed in conf/locale/config.yaml
-    where there is at least 10% reviewed translations.
+    Pull reviewed translations from all languages listed in conf/locale/config.yaml.
```

Comment:
I figured it made sense to fix the documentation, rather than the behavior, since some consumers may be relying on the current behavior.

Reply:
@mikix we did avoid using i18n transifex pull in some repos because its was not working as it was expected to work. https://github.com/edx/edx-proctoring/blob/master/Makefile#L89. --minimum-perc was used to pull atleast 1% translated translations. May be if we fix the behavior we will be able to use it in our repos we are not currently using it.

Reply:
Yes i think we should go with the --minimum-perc=10 route. Is not this better thn having empty translation files ?. i fixed the header issues when introducing the validation check for translations. and minimu-perc way was used to deal with last-translator issue. plus it helps us get rid of empty translation files.